README.md

```diff
@@ -1,64 +1,45 @@
-Hyperledger Cactus is currently undergoing a major refactoring effort to enable the desired to-be architecture which will enable plug-in based collaborative development to increase the breadth of use cases & Ledgers supported.
+Cacti is an _Incubation_ Hyperledger project, inheriting that status from Hyperledger Cactus. Information on the different stages of a Hyperledger project and graduation criteria can be found in
+the [Hyperledger Project Incubation Exit Criteria document](https://wiki.hyperledger.org/display/TSC/Project+Incubation+Exit+Criteria).
 
 ## Scope of Project
 
-As blockchain technology proliferates, blockchain integration will become an increasingly important topic in the broader blockchain ecosystem.  For instance, people might want to trade between multiple different blockchains that are run on different platforms. Hyperledger Cactus is a web application system designed to allow users to securely integrate different blockchains. It includes a set of libraries, data models, and SDK to accelerate development of an integrated services application. Our goal is to deliver a system that allows users of our code to securely conduct transactions between all of the most commonly used blockchains.
+The existence of several blockchain and distributed ledger technologies of different flavors in the market as well as networks of varying scopes and sizes built on them necessitates the need for interoperability and integration, lest we end up with a fragmented ecosystem where digital assets and the workfows (often contracts) governing them remain isolated in siloes. The solution to this is not to force all chains to coalesce (i.e., "*a single chain to rule them all*") but rather enable the networks to orchestrate transactions spanning their boundaries without sacrificing security, privacy, or governance autonomy (i.e., self-sovereignty). Hyperledger Cacti offers a family of protocols, modules, libraries, and SDKs, that can enable one network to be interoperable with, and carry out transactions directly with, another while eschewing the need for a central or common settlement chain. Cacti will allow networks to share ledger data, and exchange and transfer assets atomically, and manage identities, across their boundaries, as illustrated in the figure below.
+
+<img src="./images/cacti-vision.png">
 
-## Run the Examples
+As a fusion of two earlier systems (Cactus and Weaver) that have similar philosophies and goals, yet offer distinct mechanisms backed by differemt design and trust assumptions, Cacti offers a spectrum of seletable and configurable features for cross-network transaction orchestrations. An example illustrated below shows how distributed applications running on Fabric and Besu ledgers respectively can carry out the same set of cross-network transactions using the **Node Server** (Cactus legacy) or through **Relays** (Weaver legacy).
 
-### Supply Chain Example
+<img src="./images/tx-orchestration-modes.png">
 
```

Comment:
Could you add "the list of Supported ledgers by Cacti" as follow?  I think many people are interested in what ledgers can be supported by Cacti because Cacti is an interoperability tool.

--

## Supported ledgers by Cacti:

  - Hyperledger Besu: [connector (cactus ver.)](https://github.com/hyperledger/cacti/tree/main/packages/cactus-plugin-ledger-connector-besu)
  - Hyperledger Fabric: [connector (cactus ver.)](https://github.com/hyperledger/cacti/tree/main/packages/cactus-plugin-ledger-connector-fabric), [driver (weaver ver.)](https://github.com/hyperledger/cacti/tree/main/weaver/core/drivers/fabric-driver)
  - Hyperledger Indy: [connector (cactus ver.)](https://github.com/hyperledger/cacti/tree/main/packages-python/cactus_validator_socketio_indy)
  - Hyperledger Iroha: [connector to Iroha V1 (cactus ver.)](https://github.com/hyperledger/cacti/tree/main/packages/cactus-plugin-ledger-connector-iroha), [connector to Iroha V2 (cactus ver.)](https://github.com/hyperledger/cacti/tree/main/packages/cactus-plugin-ledger-connector-iroha2)
  - Hyperledger Sawtooth: [connector (cactus ver.)](https://github.com/hyperledger/cacti/tree/main/packages/cactus-plugin-ledger-connector-sawtooth-socketio)
  - R3 Corda: [connector (cactus ver.)](https://github.com/hyperledger/cacti/tree/main/packages/cactus-plugin-ledger-connector-corda), [driver (weaver ver.)](https://github.com/hyperledger/cacti/tree/main/weaver/core/drivers/corda-driver)
  - Go-Ethereum: [connector (cactus ver.)](https://github.com/hyperledger/cacti/tree/main/packages/cactus-plugin-ledger-connector-go-ethereum-socketio)
  - Quorum: [connector (cactus ver.)](https://github.com/hyperledger/cacti/tree/main/packages/cactus-plugin-ledger-connector-quorum)
  - Xdai: [connector (cactus ver.)](https://github.com/hyperledger/cacti/tree/main/packages/cactus-plugin-ledger-connector-xdai)
  - Ubiquity SDK: [connector (cactus ver.)](https://github.com/hyperledger/cacti/tree/main/packages/cactus-plugin-ledger-connector-ubiquity)
  - (will be added in future versions)

  - NOTE: The above "cactus ver." and "weaver ver." are the current classification on Cacti v2-alpha.  In the refactoring on Cacti v2, these classification will be rearranged to be easy for users.

Reply:
How about we add this (after removing the Weaver references) to `README-cactus.md`? `weaver/README.md` already contains a table with supported DLTs and features.

To me, this particular README is a (very) temporary and minimal file that serves as a placeholder telling users what the current code base contains and how to navigate through it. What I expect users to get from it (for the next few weeks or so until we integrate the code and workflows a bit) is that the `cactus` code is intact in the root folder and the `weaver` code is intact in the `weaver` sub folder.

As soon as we do the first round of integration, say publish packages in a common namespace, we can list various supported features. I'd like to list not just the DLTs but also various use case patterns and modules that are supported by the platform.

If we say anything more at this time in the README, I fear we are overpromising our users. The `weaver` parts are not ready for use yet as they must be ported and Github actions must be configured.

Reply:
@VRamakrishna I agree your suggestion that the current features are described in each READMEs for a while.  In the near future, I think we should rewrite the features of Cactus and Weaver on the single file.  Of course, it is very important to describe the code location of Weaver in order to spread not only Cactus but also Weaver features to new users.